From dd5aa8f848e71e39388530c017800d0bfcfcb691 Mon Sep 17 00:00:00 2001
From: Philipp Kern <phil@philkern.de>
Date: Thu, 5 May 2011 22:28:25 +0200
Subject: [PATCH 1/8] configure.ac: fix build breakage with GNU stow installed

---
 configure.ac |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 5cab4e6..ca07c53 100644
--- a/configure.ac
+++ b/configure.ac
@@ -27,11 +27,6 @@ AM_INIT_AUTOMAKE([foreign 1.9 -Wall])
 
 AC_SUBST(PACKAGE_URL, [http://0pointer.de/lennart/projects/padevchooser/])
 
-if type -p stow > /dev/null && test -d /usr/local/stow ; then
-   AC_MSG_NOTICE([*** Found /usr/local/stow: default install prefix set to /usr/local/stow/${PACKAGE_NAME}-${PACKAGE_VERSION} ***])
-   ac_default_prefix="/usr/local/stow/${PACKAGE_NAME}-${PACKAGE_VERSION}"
-fi
-
 # Checks for programs.
 AC_PROG_CXX
 AC_PROG_CC
-- 
1.7.4.4

